spring-boot-starter/autoconfigure/src/main/java/tech/cassandre/trading/bot/dto/market/TickerDTO.java

```diff
@@ -87,28 +87,4 @@ public CurrencyDTO getQuoteCurrency() {
             return null;
         }
     }
-
```

Comment:
I cannot use lomboz equals as I have my own equalsbuilder. Lomboz  makes a .equals() but on zonedDateTime the same moment is not equal if timzeone are different

Reply:
would you have different timezones on different tickers?
From what I see the timestamp is converted to system timezone for every ticker

Reply:
Some peoples are asking me to store tickers and, like for bigdecimal, the value stored in database is not necessarily the same that the value saved in Java. So the value retrieved would be different. I think it's better to have a solution in mapstruct. Do you want me to do it ?

Reply:
I don't have time in the following days, so if you can, please.
Anyway, did not knew about ZonedDateTime and just took a look:

`  @Override
    public boolean equals(Object obj) {
        if (this == obj) {
            return true;
        }
        if (obj instanceof ZonedDateTime) {
            ZonedDateTime other = (ZonedDateTime) obj;
            return dateTime.equals(other.dateTime) &&
                offset.equals(other.offset) &&
                zone.equals(other.zone);
        }
        return false;
    }`

Seems it does check equality of the offset and zone.